File: Subclass/whiteBackground.m
//
//  whiteBackground.m
//  BeFit
//
//  Created by Jon Brown on 11/10/14.
//
//

#import "whiteBackground.h"

@implementation whiteBackground

- (void)drawRect:(NSRect)dirtyRect {
    
    
    NSColor *backColor = [NSColor colorWithCalibratedRed:1.0 green:1.0 blue:1.0 alpha:1.0f];
    
    [backColor setFill];
    NSRectFill(dirtyRect);
    [super drawRect:dirtyRect];
}


@end

File: Subclass/WhiteBackground.swift
//
//  WhiteBackground.swift
//  BeFit
//
//  Created by Andrej Slegl on 11/6/19.
//

import Foundation
import Cocoa

class WhiteBackground: NSView {
    override func draw(_ dirtyRect: NSRect) {
        let backColor = NSColor(calibratedRed: 1, green: 1, blue: 1, alpha: 1)
        backColor.setFill()
        dirtyRect.fill()
        super.draw(dirtyRect)
    }
}
